### PR TITLE
Lra 434 avm create a source selection for the source input button

### PR DESCRIPTION
**Pull request by britaumich**
This is a draft pull request. 
On a room show page, a drop-down menu of available sources is displayed for a projector. If a source is changed the sendtosocket stimulus controller will fetch /send_to_room/${room_id}?operation=source_int&source=${source_id}` to send a message to the socket.
For testing, I commented SendSocketJob in the send_to_room method.
Instead of starting that job, I added the Rails.logger command to print msg in the rails log.

Change confirmation is in the stimulus controller.

### CODE REVIEW

**Comment by britaumich** [comment]
@rsmoke The last commit sends the correct value as a selected source to the websocket.
I can uncomment SendSocketJob in the rooms_controller and convert the PR from draft to active one or do you have any other comments before I do that?

**Comment by britaumich** [comment]
I merged LRA-433-avm-create-a-toggle-for-volume-level-from-1-100 into this branch because they both use the same stimulus controller and the same method in rooms_controller, and create things on the same room show page.
I uncommented   SendSocketJob in the send_to_room method.

File: app/controllers/rooms_controller.rb [hunk 1]
```diff
@@ -134,9 +134,11 @@ def send_to_room
...
     when 'mic_vol'
-      msg = "{'LSARoom': {'ShortIntegerOutputs':{'Set Mic Volume': 0}, 'Password': 'LSAPassword'}}"
+      volume = params[:volume]
+      msg = "{'LSARoom': {'ShortIntegerOutputs':{'Set Mic Volume': " + "#{volume}}, 'Password': 'LSAPassword'}}"
     when 'source_vol'
-      msg = "{'LSARoom': {'ShortIntegerOutputs':{'Set Source Volume': 0}, 'Password': 'LSAPassword'}}"
+      volume = params[:volume]
+      msg = "{'LSARoom': {'ShortIntegerOutputs':{'Set Source Volume': " + "#{volume}}, 'Password': 'LSAPassword'}}"
     when 'projector_on'
       msg = "{'LSARoom': {'Assets':{'Projector 1':{'BooleanOutputs': {'Power On': true}}}, 'Password': 'LSAPassword'}}"
```
Comment (rsmoke) [inline app/controllers/rooms_controller.rb:143]: Initially the JSON being sent for the "projector_on" and "projector_off" were hard coded to the asset name of 'Projector X". This needs to be dynamic now. @britaumich and @rsmoke discussed @britaumich  adding the change to making this dynamic. This also is on line 145

**Review by rsmoke — CHANGES_REQUESTED** [review]
change the setting of the hardcoded value for the asset 'Projector 1' to be dynamic. See inline comment

**Pull request closed without merging**